### PR TITLE
updated bit version + fixed build bug

### PR DESCRIPTION
**Pull request by amitgilad3**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/bit.rb [hunk 1]
```diff
@@ -2,15 +2,8 @@
 class Bit < Formula
   desc "Distributed Code Component Manager"
   homepage "https://www.bitsrc.io"
-  url "https://bitsrc.jfrog.io/bitsrc/bit-brew/stable/bit/0.1.12/bit-0.1.12-brew.tar.gz"
-  sha256 "122ec737cca207bba1fa9ae6e3ea22acf53cecd56b542a2a5bacd380a88bdef9"
-
-  bottle do
```
Comment (nijikon) [inline Formula/bit.rb]: Please do not remove this block.

**Review by nijikon — CHANGES_REQUESTED** [review]
Please rebase this on top of master.

File: Formula/bit.rb (hunk 1 above)
Comment (amitgilad3) [inline Formula/bit.rb]: this is an old block ,current version of bit is 0.1.19

Comment (nijikon) [inline Formula/bit.rb]: Yes but you are removing `bottle` block, this is replace by test bot automatically.

**Pull request closed without merging**